kura/examples/test/org.eclipse.kura.example.wire.logic.multiport.provider.test/pom.xml

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2017, 2020 Eurotech and/or its affiliates and others
```

Comment:
Hey @iacco, this looks like a new file (unless you changed it entirely), so the header should not contain year 2017, but only 2020?